8Knot/components/visualization.py

```
@@ -0,0 +1,93 @@
+from dash import html, dcc, Input, State, Output, callback, MATCH
+import dash_bootstrap_components as dbc
+
+
+# All-in-One Components should be suffixed with 'AIO'
+class VisualizationAIO(dbc.Card):
+    def __init__(
+        self, page: str, viz_id: str, graph_info="", class_name="", controls=None, title: str = "", id: str = None
```

Ruff (0.14.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)